src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+public class Deadline extends Task{
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + " ( by: " + by + ")";
+    }
+}
+<<<<<<< .merge_file_NUjoEu
+
+
+
+=======
+>>>>>>> .merge_file_6VnG6A
```

Comment:
???? please take care to clear the merge conflicts before pushing; it becomes especially important when you are working on your tP